src/routes/GroupSettings/GroupSettings.connector.js

```diff
@@ -24,12 +24,12 @@ import { fetchLocation } from 'components/LocationInput/LocationInput.store'
 export function mapStateToProps (state, props) {
   const slug = getRouteParam('groupSlug', state, props, false)
   const group = getGroupForCurrentRoute(state, props)
-  const canModerate = getCanModerate(state, { group })
   const currentUser = getMe(state)
   const parentGroups = group ? getParentGroups(state, { groupSlug: group.slug }) : []
+  const commonRoles = getCommonRoles(state)
```

Comment:
I only see geCommonRoles used here; I would have assumed it would be needed to display badges correctly. Is there some other way of getting the common role data for the badges? I'll have a look

Reply:
ok, I see it happening in `getRolesForGroup`, thats probably what I'm looking to port to mobile

Reply:
indeed